AirCasting/SearchAndFollow/CompleteScreen/Services/SearchAndFollowCompleteScreenService.swift

```diff
@@ -64,6 +64,11 @@ struct DefaultSearchAndFollowCompleteScreenService: SearchAndFollowCompleteScree
             case .success:
                 uiStore.accessStorage { store in
                     store.giveHighestOrder(to: session.uuid)
+                    do {
+                        try store.cardStateToggle(for: session.uuid)
+                    } catch {
+                        Log.error("Changing card state failed: \(error)")
```

Comment:
don't we want `completion(.failure(error))` here?

Reply:
I think this uiState thing is not essential so I wouldn't block following a session only because we couldn't save the card state